tests/test_ui.rs

```diff
@@ -7,19 +7,5 @@ fn ui() {
   env::set_var("DENO_TRYBUILD", "1");
 
   let t = trybuild::TestCases::new();
-  t.compile_fail("tests/compile_fail/boxed_local.rs");
-  t.compile_fail("tests/compile_fail/handle_scope_escape_lifetime.rs");
-  t.compile_fail("tests/compile_fail/handle_scope_lifetime_1.rs");
-  t.compile_fail("tests/compile_fail/handle_scope_lifetime_2.rs");
-  t.compile_fail("tests/compile_fail/handle_scope_lifetime_3.rs");
-  t.compile_fail("tests/compile_fail/handle_scope_lifetime_4.rs");
-  t.compile_fail("tests/compile_fail/try_catch_exception_lifetime.rs");
-  t.compile_fail("tests/compile_fail/try_catch_message_lifetime.rs");
-
-  // For unclear reasons rustc on Windows in Github Actions omits some
-  // diagnostic information, causing this test to fail. It might have something
-  // to do with this Rust issue: https://github.com/rust-lang/rust/issues/53081.
-  if cfg!(not(windows)) || env::var("GITHUB_ACTION").is_err() {
-    t.compile_fail("tests/compile_fail/handle_scope_escape_to_nowhere.rs");
-  }
+  t.compile_fail("tests/compile_fail/*.rs");
```

Comment:
I'm sure you checked this, but just a reminder to check that this is actually running the tests.

Reply:
It definitely does, and it is also the way we used to have it before one test had to be disabled on GHA-windows. 

Also notice that all the stderr files were updated - not something you'd expect when tests aren't running. https://github.com/denoland/rusty_v8/pull/406/files?file-filters%5B%5D=.stderr